src/domains/todo/services.ts

```diff
@@ -0,0 +1,31 @@
+const getTodoCollection = (userId: string), collection(user, ["users", userId, "todos"])
```

Comment:
It doesn't work, also it's invalid syntax
```suggestion
const getTodoCollection = (userId: string) => collection(user, ["users", userId, "todos"])
```